src/libs/CheckPDFDocument/index.native.ts

```diff
@@ -0,0 +1,11 @@
+import {NativeModules} from 'react-native';
+import type CheckPDFDocument from './types';
+
+const checkPDFDocument: CheckPDFDocument = {
+    isValidPDF: (path, callback) =>
+        new Promise(() => {
+            NativeModules.CheckPDFDocument.checkPdf(path, callback);
+        }),
+};
```

Comment:
```suggestion
const checkPDFDocument: CheckPDFDocument = {
    isValidPDF: (path) =>
        new Promise((resolve) => {
            NativeModules.CheckPDFDocument.checkPdf(path, resolve);
        }),
};
```

What about making use of Promises? Also please declare types for `CheckPDFDocument` module like it was done for others [here](https://github.com/Expensify/App/blob/main/src/types/modules/react-native.d.ts#L359).